analyzer/runtime/runtime.go

```diff
@@ -88,14 +88,14 @@ func NewRuntimeAnalyzer(nodeCfg config.NodeConfig, cfg *config.BlockBasedAnalyze
 		Source: client,
 	}
 
-	qf := analyzer.NewQueryFactory(strcase.ToSnake(nodeCfg.ChainID), emerald.String())
+	qf := analyzer.NewQueryFactory(strcase.ToSnake(nodeCfg.ChainID), sapphire.String())
 
 	return &Main{
 		cfg:     ac,
 		qf:      qf,
 		target:  target,
-		logger:  logger.With("analyzer", EmeraldDamaskAnalyzerName),
-		metrics: metrics.NewDefaultDatabaseMetrics(EmeraldDamaskAnalyzerName),
+		logger:  logger.With("analyzer", SapphireDamaskAnalyzerName),
+		metrics: metrics.NewDefaultDatabaseMetrics(SapphireDamaskAnalyzerName),
```

Comment:
probably split into two key-values. analyzer -> runtime, runtime -> that parameter

Reply:
now we're doing analyzer -> that parameter